lib/lattice_observer/observed/event_processor.ex

```diff
@@ -229,35 +249,75 @@ defmodule LatticeObserver.Observed.EventProcessor do
 
     l = record_host(l, source_host, labels, stamp, friendly_name)
 
+    # legacy heartbeat has a list for the actors field...
+    # default to "new format" if this field is missing
```

Comment:
Suggestion: specify that this can be removed once we're far enough in the future that all hosts can be assumed to be v0.60+